it/lessons/basics/basics.md

```diff
@@ -174,6 +211,8 @@ iex> not 42
 ** (ArgumentError) argument error
 ```
 
+Nota: Le operaioni booleane `and` e `or` in Elixir vengono mappate a `andalso` e `orelse` in Erlang.
```

Comment:
Fix `operaioni` with `operazioni`.

Also: `mappate` is not proper italian, is a sort of programmer neologism derived from english term. Maybe it's ok here after all, just pointing it out.

Reply:
fixed in efe71e5a7017e4a5602d069b368c9c56d9880e4b